File: ios/filtering-content/GetSubpages.swift
// DocSection: filtering_get_subpages
// Gets pages linking travel insurance as their subpage.
let customQuery = "items?elements.subpages[contains]=travel_insurance"
// Gets pages linking at least travel insurance, car insurance, or both as their subpage.
// let customQuery = "items?elements.subpages[any]=travel_insurance,car_insurance"

client.getItems(modelType: Page.self, customQuery: customQuery) { (isSuccess, itemsResponse, error) in
    if isSuccess {
        if let pages = itemsResponse?.items {
            // Use your items here
        }
    } else {
        if let error = error {
            print(error)
        }
    }
}
// EndDocSection

File: java/filter-content/GetSubpages.java
// DocSection: filtering_get_subpages
// Gets pages linking travel insurance as their subpage.
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterContains("elements.subpages", "travel_insurance")
    .build()
);

// Gets pages linking at least travel insurance, car insurance, or both as their subpage.
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterAny("elements.subpages", "travel_insurance", "car_insurance")
    .build()
);
// To use the code for Android projects, see http://kontent.ai/learn/android
// EndDocSection